app/main.py

```diff
@@ -12,7 +34,103 @@
     allow_headers=["*"],
 )
 
-
 @app.get("/")
 async def hello():
-    return {"message": "Hello World"}
\ No newline at end of file
+    return {"message": "Hello World"}
+
+@app.post("/api")
+async def search_izakaya(izakaya_search_request: Prompt) -> List[Izakaya] | dict:
+    izakaya_info_list = []         # type: List[Izakaya]
+    fieldlist = []              # type: list[str]
+
+    current_location = Coordinate(
+        coordinate=izakaya_search_request.current_coodinate     # type: tuple[float, float]
+    )
```

Comment:
モデルをレビューで指摘したように`Coordinate`クラスでラップするとわざわざ型変換をしなくて済みます
```suggestion
        current_coordinate=izakaya_search_request.current_coodinate
```

Reply:
ほかのレビューで指摘されてたように、
```
class Prompt(BaseModel):
    current_coodinate: Optional[Coordinate] = None
    prompt:  str  
```
`current_coodinate`の型をCoordinate型に直したから、型変換いらなくなった！
指摘感謝！